test/e2e/load/load.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2021 ScyllaDB
+
+package load
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/table"
+)
+
+type Load struct {
+	session  gocqlx.Session
+	table    *table.Table
+	expected []*test
+}
+
+type test struct {
+	Id   int
+	Data string
+}
+
+func NewLoad(hosts ...string) (*Load, error) {
```

Comment:
Given this lives in the e2e package (so it's dedicated) and it's used in many tests, it might be better if the call site is a one liner taking a ScyllaCluster and figuring out the hosts and other stuff internally.

Reply:
verifyScyllaCluster now takes an optional func as a parameter